modules/blockapis/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@bitgo/blockapis",
+  "description": "Access public block explorer APIs for a variety of coins",
```

Comment:
This is not quite true yet: the package defines the interfaces `AddressApi`, `TransactionApi` and `UtxoApi`. 

* The `BlockchairApi` class implements all three interfaces for all utxo mainnet networks plus bitcoin testnet.
* The `BlockstreamApi` class implements all three interfaces for btc mainnet and testnet

However it should be quite simple to implement the `TransactionApi` and `AddressApi` for `BlockchairApi` for other coins as well.